pandas/tests/frame/test_replace.py

```diff
@@ -466,6 +466,15 @@ def test_regex_replace_dict_nested(self):
         assert_frame_equal(res3, expec)
         assert_frame_equal(res4, expec)
 
+    def test_regex_replace_dict_nested_non_first_character(self):
+        # Ensures that the regular expression is replaced
```

Comment:
Minor but just leave a reference to the issue number in the comments, so `# GH 25259`

Reply:
@WillAyd Do you mean that I remove the comment and replace it only with the issue number, namely `# GH 25259`?